### PR TITLE
Changed try-except in pytest_factoryboy/__init__.py to catch all exce…

### PR DESCRIPTION
**Pull request by amakhnach**
…ptions.

### CODE REVIEW

File: pytest_factoryboy/__init__.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
         register.__name__,
         LazyFixture.__name__,
     ]
-except ImportError:  # pragma: no cover
-    # avoid import errors when only __version__ is needed (for setup.py)
+except:  # pragma: no cover
```
Comment (bubenkoff) [inline pytest_factoryboy/__init__.py]: Catch Exception instead
And issue a warning with it via python warnings module

**Comment by bubenkoff** [comment]
TypeError: expected string or buffer
Use trackeback.format_exc

**Comment by bubenkoff** [comment]
Also please rebase to master

File: pytest_factoryboy/__init__.py [hunk 2]
```diff
@@ -9,6 +11,5 @@
         register.__name__,
         LazyFixture.__name__,
     ]
-except ImportError:  # pragma: no cover
-    # avoid import errors when only __version__ is needed (for setup.py)
-    pass
+except Exception, e:  # pragma: no cover
```
Comment (olegpidsadnyi) [inline pytest_factoryboy/__init__.py]: as e

**Comment by bubenkoff** [comment]
that's fixed now

**Pull request closed without merging**